lib/cocina/models/collection.rb

```diff
@@ -16,6 +16,8 @@ class Collection < Struct
 
       # Subschema for access concerns
       class Access < Struct
+        attribute :access, Types::String.default('dark')
+                                        .enum('world', 'stanford', 'location-based', 'citation-only', 'dark')
```

Comment:
I know these are specified in the maps., but what are location-based and citation-only? Are these actually supported in DOR?

Reply:
They are presently supported in DOR. location-based means "In a reading room" and citation-only is just acknowledging we hold it in purl, but not showing the content.